pkg/cloud/aws/services/ec2/vpc.go

```diff
@@ -89,10 +89,19 @@ func (s *Service) createVPC() (*v1alpha1.VPCSpec, error) {
 		return nil, errors.Wrap(err, "failed to create vpc")
 	}
 
+	// Cannot set both attributes at the same time.
 	attrInput := &ec2.ModifyVpcAttributeInput{
 		VpcId:              out.Vpc.VpcId,
 		EnableDnsHostnames: &ec2.AttributeBooleanValue{Value: aws.Bool(true)},
-		EnableDnsSupport:   &ec2.AttributeBooleanValue{Value: aws.Bool(true)},
+	}
+
+	if _, err = s.scope.EC2.ModifyVpcAttribute(attrInput); err != nil {
+		return nil, errors.Wrap(err, "failed to set vpc attributes")
+	}
+
+	attrInput = &ec2.ModifyVpcAttributeInput{
+		VpcId:            out.Vpc.VpcId,
+		EnableDnsSupport: &ec2.AttributeBooleanValue{Value: aws.Bool(true)},
```

Comment:
Seems we can avoid this call during creation given that it defaults to `true` https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html

Reply:
Yep, it defaults to true, but I wanted to keep the same behavior of the existing code